MediaBrowser.Providers/Music/AlbumMetadataService.cs

```diff
@@ -106,6 +136,78 @@ private ItemUpdateType SetArtistsFromSongs(MusicAlbum item, IEnumerable<Audio> s
             return updateType;
         }
 
+        private ItemUpdateType SetAlbumFromSongs(MusicAlbum item, IReadOnlyList<Audio> songs)
+        {
+            var updateType = ItemUpdateType.None;
+
+            var musicbrainzAlbumIds = songs
+                .Select(i => i.GetProviderId(MetadataProvider.MusicBrainzAlbum))
+                .GroupBy(i => i)
+                .OrderByDescending(g => g.Count())
+                .Select(g => g.Key)
+                .ToArray();
+
+            var musicbrainzAlbumId = item.GetProviderId(MetadataProvider.MusicBrainzAlbum);
+            if (!String.IsNullOrEmpty(musicbrainzAlbumIds[0])
```

Comment:
What if it's empty? Can that happen?

Reply:
I think it can be set to an empty string.

Reply:
Should not be a problem anymore with the function now handling this.